bpe/test/encoder_test.py

```diff
@@ -125,7 +125,8 @@ def test_inverse_transform():
         'almost nothing should be recognized {unk} let {unk} s see ...'.format(unk=UNK)
 
     assert transform("Vizzini: He didn't fall? INCONCEIVABLE!") == \
-        "vizzini {unk} he didn {unk} t fall {unk} inconceivable {unk}".format(unk=UNK)
+        "{unk}izzini {unk} {unk}e didn {unk} t fall {unk} " \
+        "I{unk}CO{unk}C{unk}I{unk}{unk}{unk}{unk}{unk} {unk}".format(unk=UNK)
```

Comment:
Hmmm, this is a lot of UNKs for common letters.

Reply:
Yeah - it does look ugly 😓 

Although that's because those letters are absent from `test_corpus`:
https://github.com/soaxelbrooke/python-bpe/blob/54e68a9ae47a3538244259be3863b10add8d2bcc/bpe/test/encoder_test.py#L9

Reply:
Ah, this is happening due to the default non-lowercasing.